events/class_models/event_models.py

```diff
@@ -31,8 +31,18 @@ def get_all_user_day_meetings(self, user_id, date):
 
         return self.__date_filter(
             self.get_all_user_meetings(user_id),
-            start_year=year, start_month=month,
-            start_day=day
+            start_year=year, start_month=month, start_day=day,
+            end_year=year, end_month=month, end_day=day
+        )
+
+    def get_all_user_day_events(self, user_id, date):
+        """ get only the events for the user at the date given """
+        year, month, day = date.year, date.month, date.day
```

Comment:
In my opinion, we should also add a query for getting all user's events in a given month. 

In addition, can you explain please the purpose of the queries which find meetings?

Reply:
done. thanks 😄

Reply:
These queries are for a later stage in our development.

I think that a participant will want to hide irrelevant meetings that are not set yet
and by so we will need a way to differentiate between a meeting and an event.
In addition, I think that the main page should use get user month events and do not need to get all the events like the page render at this time.
And lastly, I think that a different design for a meeting that is not set is needed to be distinguished between an event and a meeting and so we will need to get the relevant meetings only and add them to a CSS class.  
But great question non the less 😄

** a design like in this picture below,
   as you can see there is a diffrence between an event and an unregistered meeting
![image](https://user-images.githubusercontent.com/63960808/167946923-50c4c5ae-58e3-40c2-93fe-e55854007188.png)

Reply:
Good thinking. I agree with the design you mentioned, and the queries can definitely help with future features. 😄